cmd/kfulint/testdata/long-chain/checker_tests.go

```diff
@@ -24,4 +40,10 @@ func sample() {
 	case b.a.b.c.d.e.p3:
 	case b.a.b.c.d.e.p4:
 	}
+
+	switch a {
+	case b.a.b.c.d.e.p1:
+	case b.a.b.c1.d.e.p1:
+	case b.a.b.c2.d.e.p1:
```

Comment:
I guess simpler sequences like `b.a.b` are not reported too, so it's OK.
Thank you.

Reply:
We should either add a todo or create an issue to add more repeated long chains like `a.b[i][j].c` to make sure that all things we like to detect are reported.
(For now, it's good enough to match only selector expressions.)